packages/devtools-launchpad/src/components/Sidebar.js

```diff
@@ -19,6 +21,35 @@ const Sidebar = React.createClass({
     onSideBarItemClick: React.PropTypes.func.isRequired
   },
 
+  renderTitle(title) {
+
+    return dom.div({className: "title-wrapper"},
+                    dom.h1({}, "Debugger"),
+                    dom.div({className: "launchpad-container"},
+                            dom.div({className: "launchpad-container-icon", dangerouslySetInnerHTML: {__html: rocketSvg }}),
```

Comment:
nice - i wouldn't have thought of this. I don't think we have a better way :)

This might be an alternative, but i'm not sure:
https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/Breakpoint.js#L10-L22

either way... this is cool for our usecase

Reply:
Yep, also think it's good for now :)